src/main/java/info/ronjenkins/slf4bukkit/AnsiColorMapper.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2016 Ronald Jack Jenkins Jr.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package info.ronjenkins.slf4bukkit;
+
+import com.google.common.collect.ImmutableMap;
+
+import org.bukkit.ChatColor;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Attribute;
+
+import java.util.Map;
+
+/**
+ * Maps {@link ChatColor} values to their JAnsi equivalents.
+ *
+ * <p>This class might not always be instantiable as jansi might not be present at runtime.</p>
+ *
+ * @author Ronald Jack Jenkins Jr.
+ */
+final class AnsiColorMapper implements ColorMapper {
+
+  AnsiColorMapper() throws Throwable {
```

Comment:
I don't think `throws Throwable` is necessary. Because the constructor is no-op, I think only subclasses of `Error` could occur, and `Error` is unchecked so it doesn't have to be declared in the method signature.